src/types/PaymentMethod.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ *
+ */
+
+import * as t from "io-ts";
+import { string } from "io-ts";
+import { enumType } from "italia-ts-commons/lib/types";
+
+export enum PaymentMethodEnum {
+  BANK_ACCOUNT = "BANK_ACCOUNT",
+  CREDIT_CARD = "CREDIT_CARD",
+  OTHER = "OTHER"
```

Comment:
do we need `OTHER`? I cannot think of a use case where we want to deal with unknown payment method types